examples/idv/views/pages/index.ejs

```diff
@@ -1,3 +1,7 @@
 <%- include('layout/header'); -%>
+<div style="font-size: 18px; margin-left: 300px">
+    <span>Session ID:</span> <span><%= sessionId %></span>
+</div>
+<a style="font-size: 18px; margin-left: 300px" href="/config" target="_blank">View session configuration</a>
```

Comment:
Is that on purpose that the href is not including the `sessionID` query params ?

Reply:
Yeah, the sessionID was not needed in the href.

Reply:
Actually added the query param.